### DISCUSSION TITLE
Quick Installer Questions

### DISCUSSION THREAD

**Discussion by zubenubi** [RaspAP/raspap-webgui#1102] [Q&A] [answered]
Hi All,

I have an existing pi on which I would like to install raspap without wiping it entirely. I went through the quick installer and it seems to have made changes that are standard for raspap but break my existing configuration. The Pi is acting as a router using just hostapd without a UI to manage the AP - which is the reason for trying to use raspap. My Questions:

-My existing wlan0 was serving out addresses on the 192.168.111.0/24 subnet with a gateway at 192.168.111.1. I see that raspap uses the 10.3.141.0/24 with wlan itself at 10.3.141.1. What would it require to adjust the settings to get back to using 192.168.111.1?

-My eth0 was on the 10.20.30.0 subnet as it is plugged to an existing internet connection. Where do I set this?

-Does the iptables instruction referencing 192.168.50.0 need to be modified to 10.20.30.0?

-Does raspap have the ability to listen on 443 exclusively (not 80) with the new cert feature?

Thanks In Advance for any answers to the above

**Comment by billz** [comment]
All of these questions are covered in the docs. 

> My existing wlan0 was serving out addresses on the 192.168.111.0/24 subnet with a gateway at 192.168.111.1. I see that raspap uses the 10.3.141.0/24 with wlan itself at 10.3.141.1. What would it require to adjust the settings to get back to using 192.168.111.1?

Refer to https://docs.raspap.com/ap-basics/#connecting-clients

> My eth0 was on the 10.20.30.0 subnet as it is plugged to an existing internet connection. Where do I set this?

RaspAP does not touch your eth0 config. You may, however, view or change its settings in the **DHCP Server** menu.

> Does raspap have the ability to listen on 443 exclusively (not 80) with the new cert feature?

The cert feature is not new. Re: ports see https://docs.raspap.com/ssl-manual/#locally-trusted-certificates
You can also set the web port on the **System > Advanced** tab.

> Does the iptables instruction referencing 192.168.50.0 need to be modified to 10.20.30.0?

Discussed in detail here https://docs.raspap.com/manual/#routing-and-ip-masquerading

> I went through the quick installer and it seems to have made changes that are standard for raspap but break my existing configuration. 

Nothing unusual here. Again, described multiple times in the docs. For example https://docs.raspap.com/ap-basics/#reverting-to-base-settings

The documentation is here to help you https://docs.raspap.com/

**Comment by zubenubi** [comment]
Thanks for the hints.

Can you tell me if its possible to change the underlying files for raspap (rather than doing so through the UI)? I would like to automate the configuration using an ansible script and I am familiar with making changes to txt files as necessary.

Thanks

**Comment by billz** [comment, marked as answer]
Take a look at https://github.com/RaspAP/raspap-ansible 😉

**Reply by zubenubi** [reply]
Thanks.
Another question. 

If I make changes to files in /var/www/html/config what do I need to do to see those changes reflected? Do I need to restart php or some other service?

**Reply by billz** [reply]
Changes to these files are reflected immediately, without restarting any services. 

Take note that `config/defaults.json` is copied to `/etc/raspap/networking` during the install and subsequently read from this location.
